sdk/mixedreality/Azure.MixedReality.RemoteRendering/README.md

```diff
@@ -0,0 +1,386 @@
+
+### Install the package
+
+Install the Azure Mixed Reality ARR client library for .NET using one of the following methods.
+
+From Visual Studio Package Manager:
+
+```powershell
+Install-Package Azure.MixedReality.RemoteRendering
+```
+
+From .NET CLI
+
+```dotnetcli
+dotnet add package Azure.MixedReality.RemoteRendering
+```
+
+Add a package reference:
+
+```xml
+<PackageReference Include="Azure.MixedReality.RemoteRendering" Version="1.0.0-preview.1" />
+```
+
+### Prerequisites
+
+You will need an [Azure subscription](https://azure.microsoft.com/free/) and an [Azure Remote Rendering account](https://docs.microsoft.com/azure/remote-rendering/how-tos/create-an-account) to use this package.
+
+### Authenticate the client
+
+Constructing a remote rendering client requires an authenticated account, and a remote rendering endpoint.
+A RemoteRenderingAccount object is constructed from an accountId and an account domain.
+For an account created in the eastus region, the account domain will have the form "eastus.mixedreality.azure.com".
+There are several different forms of authentication:
+
+- Account Key authentication
+  - Account keys enable you to get started quickly with using Azure Remote Rendering. But before you deploy your application
+    to production, we recommend that you update your app to use Azure AD authentication.
+- Azure Active Directory (AD) token authentication
+  - If you're building an enterprise application and your company is using Azure AD as its identity system, you can use
+    user-based Azure AD authentication in your app. You then grant access to your Azure Remote Rendering accounts by using
+    your existing Azure AD security groups. You can also grant access directly to users in your organization.
+  - Otherwise, we recommend that you obtain Azure AD tokens from a web service that supports your app. We recommend this
+    method for production applications because it allows you to avoid embedding the credentials for access to Azure Spatial
+    Anchors in your client application.
+
+See [here](https://docs.microsoft.com/azure/remote-rendering/how-tos/authentication) for detailed instructions and information.
+
+In all the following examples, the client is constructed with a `remoteRenderingEndpoint` Uri object.
+The available endpoints correspond to regions, and the choice of endpoint determines the region in which the service performs its work.
+An example is `https://remoterendering.eastus2.mixedreality.azure.com`.
+
+> [!Note] For converting assets, it is preferable to pick a region close to the storage containing the assets.
+
+> [!Note] For rendering, it is strongly recommended that you pick the closest region to the devices using the service. 
+> The time taken to communicate with the server impacts the quality of the experience.
+
+#### Authenticating with account key authentication
+
+Use the `AccountKeyCredential` object to use an account identifier and account key to authenticate:
+
+```csharp
```

Comment:
please use the embeddable snippets for code samples. See https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#updating-sample-snippets